fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/BaseProvider.java

```diff
@@ -90,5 +122,13 @@ public void clearCache() {
     cache.invalidate(CACHE_KEY);
   }
 
+  protected Set<R> populateCache() throws ProviderException {
```

Comment:
Probably want to make this private unless there's a good reason subclasses need to meddle with the interceptors.